docs/core/install/windows.md

```diff
@@ -388,6 +388,40 @@ Microsoft provides images that are tailored for specific scenarios. For example,
 
 For more information about using .NET in a Docker container, see [Introduction to .NET and Docker](../docker/introduction.md) and [Samples](https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md).
 
+## Troubleshooting
+
+After installing .NET, you may run into problems trying to run .NET. This section collects those common problems and provides solutions.
+
+- [Can't find/run/locate .NET after installing](#cant-findrunlocate-net-after-installing)
+
+### Can't find/run/locate .NET after installing
+
+Most likely you've installed both the x86 (32-bit) and x64 (64-bit) versions of .NET. This is causing a conflict because when you run the `dotnet` command it's resolving to the x86 version when it should resolve to the x64 version. This is usually fixed by adjusting the `%PATH%` variable to resolve the x64 version first.
```

Comment:
```suggestion
Most likely you've installed both the x86 (32-bit) and x64 (64-bit) versions of the .NET SDK. This is causing a conflict because when you run the `dotnet` command it's resolving to the x86 version when it should resolve to the x64 version. This is usually fixed by adjusting the `%PATH%` variable to resolve the x64 version first.
```